### PR TITLE
add dockerfile for librari api

### PR DESCRIPTION
**Pull request by Nguyen-Devid**

### CODE REVIEW

File: library_api/dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . ./
+
+RUN go build -o main .
+
+EXPOSE 8000
+
+CMD ["./main"]
```
Comment (diabeticMike) [inline library_api/dockerfile]: use `docker inspect [image name]` to check how much your image weight, then use multistage build and check it again -> you will notice huge memory improvement

**Review by diabeticMike — CHANGES_REQUESTED** [review]
Overall good but try to use multistage builds

File: library_api/dockerfile (hunk 1 above)
Comment (Nguyen-Devid) [reply library_api/dockerfile]: after multi-stage build 
image

Comment (diabeticMike) [reply library_api/dockerfile]: image

**Review by diabeticMike — APPROVED** [review]

**Pull request merged**